File: lib/models/analista_model.dart
class Analista {
  final int codigoAnalista;
  final String nome;
  final String cargo;
  final String email;
  final String telefone;

  Analista({
    required this.codigoAnalista,
    required this.nome,
    required this.cargo,
    required this.email,
    required this.telefone,
  });

  Map<String, dynamic> toJson() {
    return {
      'codigoAnalista': codigoAnalista,
      'nome': nome,
      'cargo': cargo,
      'email': email,
      'telefone': telefone
    };
  }

  factory Analista.fromJson(Map<String, dynamic> json){
    return Analista(
      codigoAnalista: json['codigoAnalista'],
      nome: json['nome'],
      cargo: json['cargo'],
      email: json['email'],
      telefone: json['telefone'], 
    );
  }
}

File: back-end/src/main/java/br/com/axi/stack_api/model/AnalistaModel.java
package br.com.axi.stack_api.model;

import jakarta.persistence.*;
import lombok.Data;

@Data
@Entity
@Table(name = "Analista")
public class AnalistaModel {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long codigoAnalista;

    private String nome;
    private String cargo;
    private String email;
    private String telefone;
}
